packages/ic-sha3/src/lib.rs

```diff
@@ -0,0 +1,110 @@
+//! SHA-3 and related hash functions.
+//! Currently supported are:
```

Comment:
This list seems outdated, as now also `Shake256` is supported.

How about actually removing this list, because the documentation will anyway show what is supported.

If you want to keep the description of Keccak-256, i.e., constant 256-bit output and not the same as SHA3-256, I'd move it to the Rustdoc of the `Keccak256` struct.

Reply:
> How about actually removing this list, because the documentation will anyway show what is supported.

makes sense, done.

> If you want to keep the description of Keccak-256, i.e., constant 256-bit output and not the same as SHA3-256, I'd move it to the Rustdoc of the Keccak256 struct.

I moved the 2 lines to the `Keccak256` struct as suggested.